diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -138,12 +143,31 @@ fun ASTNode.isFollowedByNewline() =
             it.treeNext.elementType == WHITE_SPACE && it.treeNext.text.contains("\n")
         } ?: false
 
+/**
+ * This function is similar to isFollowedByNewline(), but there may be a comment after the node
+ */
+fun ASTNode.isFollowedByNewlineWithComment() =
+        parent({ it.treeNext != null }, strict = false)?.let { astNode ->
+            astNode.treeNext.run {
```

Comment:
```suggestion
        parent({ it.treeNext != null }, strict = false)
            ?.treeNext.run {
```
I don't think you need this additional scope here.